src/Components/CascadingEndTimesBanner.tsx

```diff
@@ -13,7 +13,9 @@ const CascadingEndTimesBanner: React.FC<CascadingEndTimesBannerProps> = ({
 }) => {
   const helpArticleLink = getENV("CASCADING_AUCTION_HELP_ARTICLE_LINK")
 
-  if (!sale.cascadingEndTimeIntervalMinutes) {
+  const saleIsClosed = !!sale.endedAt
```

Comment:
If there's a boolean `isClosed` field, why opt for coercing a date field `endedAt` and then aliasing the coerced value to local variable name representing something we can just return from the backend? (1:2)

Reply:
If we can't trust `isClosed` (which is used all over many of our codebases if one does a quick search) then there are larger problems! Its minor, but still something to take to heart as a pattern to avoid.